tests/testthat/test_survey_statistics.r

```diff
@@ -1,11 +1,6 @@
 context("Quick tests for summary stats (ratio / quantile)")
 
-# TODO: switch to improved svyquantile
-if (utils::packageVersion("survey") >= "4.1") {
-  svyq_func <- get("oldsvyquantile", asNamespace("survey"))
-} else {
-  svyq_func <- survey::svyquantile
-}
+svyq_func <- survey::svyquantile
```

Comment:
Can you change the places that used `svyq_func` to just be svyquantile now (and then delete this line)? (Like you did in the other test file)

Reply:
Another one resolved by https://github.com/gergness/srvyr/pull/124/commits/f1a4cd92f8c8c03e9766495b5e0299ef6fc77d7b.